packages/common/src/components/Button/index.tsx

```diff
@@ -21,6 +21,7 @@ type Props = {
 const Button: FunctionComponent<Props> = ({ style = {}, ...props }) => {
   // Link
   if (typeof props.to === 'string') {
+    // @ts-ignore
     return <LinkButton {...props} style={style} to={props.to} />;
```

Comment:
We were using react-router-dom@5 with @types/react-router-dom@4, updating the types brought up this error :)

Assuming it was already working till now, Added ts-ignore to ignore the type warning that should have been there forever :)